src/index.tsx

```diff
@@ -134,7 +134,18 @@ function doPick<OS extends SupportedPlatforms>(
     throw new TypeError('Invalid copyTo option: ' + options.copyTo)
   }
 
-  return RNDocumentPicker.pick(options)
+  return RNDocumentPicker.pick(options).then((pickerDocuments) => {
+    if (Platform.OS === 'ios') {
+      return pickerDocuments.map((pickerDocument) => ({
```

Comment:
This information might be a minor improvement:

```js
return RNDocumentPicker.pick(options)
    .then((pickerDocuments) => {
        if (Platform.OS === 'ios') {
            pickerDocuments.forEach((document) => {
                document.uri = decodeURIComponent(document.uri);
                if (document.fileCopyUri) {
                    document.fileCopyUri = decodeURIComponent(document.fileCopyUri);
                }
            });
        }
        return pickerDocuments;
    })
```

Because:

1. Your existing solution will inject `fileCopyUri: null` if a `pickerDocument` previously had no `fileCopyUri` key, which may not be a big deal but might be a change from the existing behavior.
1. It's slightly more performant because it does not use the spread operator to copy all the documents unnecessarily (disclaimer: I don't know how big these objects are or if that performance difference would be negligible)

Reply:
Thank you for your suggestion, @roryabraham. This makes sense. I'm pushing a change to incorporate it.